ui/modules/connections/connections.js

```diff
@@ -257,16 +257,16 @@ function loadConnections() {
   let connectionSelected = false;
   API.callConnectionsAPI('listConnections', (connections) => {
     connections.forEach((connection) => {
-      const id = API.env() === 'things' ? connection.id : connection;
+      const id = API.env() === 'ditto_2' ? connection : connection.id;
       const row = dom.tbodyConnections.insertRow();
       row.id = id;
-      if (API.env() === 'things') {
-        row.insertCell(0).innerHTML = connection.name;
-      } else {
+      if (API.env() === 'ditto_2') {
         API.callConnectionsAPI('retrieveConnection', (dittoConnection) => {
-          row.insertCell(0).innerHTML = dittoConnection.name;
-        },
-        id);
+           row.insertCell(0).innerHTML = dittoConnection.name;
+         },
+         id);
```

Comment:
Indentation was proclaimed by eslint
```suggestion
         row.insertCell(0).innerHTML = dittoConnection.name;
       },
       id);
```